### ISSUE TITLE
Devtools performance example tests is not working

### ISSUE THREAD

**Issue by pmerwin** [webdriverio/webdriverio#4055]
session.off is not a function error when trying to capture performance via devtools:

**Environment (please complete the following information):**
 - **WebdriverIO version:**  5.9.6
 - **Mode:** WDIO Testrunner
 - **If WDIO Testrunner, running sync/async: sync
 - **Node.js version:** [8.10
 - **NPM version:** [e.g. 5.8.0]
 - **Browser name and version:** chrome Version 74.0.3729.169
 - **Platform name and version:**OSX 
 - **Additional wdio packages used (if applicable):** @wdio/devtools-service, chromdriver
**Config of WebdriverIO**
An example of how you configured WebdriverIO config

**Describe the bug**
Following the example at:
https://webdriver.io/docs/devtools-service.html#access-puppeteer-instance
produces an error : 
```
session.off is not a function
[chrome  mac os x #0-0] TypeError: session.off is not a function
[chrome  mac os x #0-0]     at cancel (/Users/pmerwin/Projects/qa/WDIO-5/lib-webdriverio-nodejs-v2/node_modules/@wdio/devtools-service/build/gatherer/trace.js:341:17)
```

**To Reproduce**
config:
```
const merge = require('deepmerge');
const wdioBaseConf = require('./base');
exports.config = merge(wdioBaseConf.config, {
    capabilities: [{
        browserName: 'chrome',
        platformName: 'MAC',
        'goog:chromeOptions': {
            w3c: true
        }
    }],
    port: 9515,
    path: '/',
    // ...
    services: ['chromedriver', 'devtools'], // wdio, set this in package.json for chromedriver
    mochaOpts: {
        ui: 'bdd',
        compilers: ['js:@babel/register'],
        timeout: 120000,
        bail: true
    },
    reporters: ['spec']
});
```
with spec:

```
const assert = require('assert');

describe('JSON.org page', () => {
    before(() => {
        browser.enablePerformanceAudits();
    });

    it('should load within performance budget', () => {
        browser.url('http://json.org');

        let metrics = browser.getMetrics();
        assert.ok(metrics.speedIndex < 1500); // check that speedIndex is below 1.5ms

        let score = browser.getPerformanceScore(); // get Lighthouse Performance score
        assert.ok(score >= 0.99); // Lighthouse Performance score is at 99% or higher

        $('=Esperanto').click();

        metrics = browser.getMetrics();
        assert.ok(metrics.speedIndex < 1500);
        score = browser.getPerformanceScore();
        assert.ok(score >= 0.99);
    });

    after(() => {
        browser.disablePerformanceAudits();
    });
});
```
**Expected behavior**
Test should run and deliver metrcis data but fails with an error

**Log**
```
2019-06-04T18:06:46.657Z INFO @wdio/local-runner: Run worker command: run
2019-06-04T18:06:46.960Z INFO webdriver: [POST] http://127.0.0.1:9515/session
2019-06-04T18:06:46.960Z INFO webdriver: DATA { capabilities: 
   { alwaysMatch: 
      { browserName: 'chrome',
        platformName: 'MAC',
        'goog:chromeOptions': [Object] },
     firstMatch: [ {} ] },
  desiredCapabilities: 
   { browserName: 'chrome',
     platformName: 'MAC',
     'goog:chromeOptions': { w3c: true } } }
2019-06-04T18:06:48.669Z INFO @wdio/devtools-service:DevToolsDriver: Connect to http://localhost:55949
2019-06-04T18:06:48.715Z INFO @wdio/devtools-service: Connected to Chrome on localhost:55949
2019-06-04T18:06:49.275Z INFO webdriver: COMMAND enablePerformanceAudits()
2019-06-04T18:06:49.278Z INFO webdriver: RESULT undefined
2019-06-04T18:06:49.282Z INFO @wdio/devtools-service:TraceGatherer: Start tracing frame with url http://json.org/
2019-06-04T18:06:49.390Z INFO @wdio/devtools-service:TraceGatherer: Waiting for CPU to become idle
2019-06-04T18:06:49.400Z INFO webdriver: COMMAND navigateTo("http://json.org/")
2019-06-04T18:06:49.401Z INFO webdriver: [POST] http://127.0.0.1:9515/session/673ea9b7b70adce4e662780c094bd4e7/url
2019-06-04T18:06:49.401Z INFO webdriver: DATA { url: 'http://json.org/' }
2019-06-04T18:06:49.527Z INFO @wdio/devtools-service:TraceGatherer: Ignore navigated frame with url about:blank
2019-06-04T18:06:49.863Z INFO @wdio/devtools-service:TraceGatherer: Driver CPU has been idle for 0 ms
2019-06-04T18:06:49.863Z INFO @wdio/devtools-service:TraceGatherer: Driver CPU has been idle for 0 ms
2019-06-04T18:06:49.875Z INFO @wdio/devtools-service:TraceGatherer: Waiting on DomContentLoaded
2019-06-04T18:06:50.704Z INFO @wdio/devtools-service:TraceGatherer: Page load detected: http://json.org/, set frameId 5220C4EB6EB7F7B4A95EB13D9177CC1D, set loaderId A8D058D89051AD1221679060CAAE9D48
2019-06-04T18:06:50.705Z INFO @wdio/devtools-service:TraceGatherer: Waiting on DomContentLoaded
2019-06-04T18:06:50.713Z INFO @wdio/devtools-service:TraceGatherer: Waiting on DomContentLoaded
2019-06-04T18:06:50.713Z INFO @wdio/devtools-service:TraceGatherer: Waiting on DomContentLoaded
2019-06-04T18:06:50.713Z INFO @wdio/devtools-service:TraceGatherer: Waiting on DomContentLoaded
2019-06-04T18:06:50.714Z INFO @wdio/devtools-service:TraceGatherer: Waiting on DomContentLoaded
2019-06-04T18:06:50.714Z INFO @wdio/devtools-service:TraceGatherer: Waiting on DomContentLoaded
2019-06-04T18:06:50.714Z INFO @wdio/devtools-service:TraceGatherer: Waiting on DomContentLoaded
2019-06-04T18:06:50.714Z INFO @wdio/devtools-service:TraceGatherer: Waiting on DomContentLoaded
2019-06-04T18:06:50.714Z INFO @wdio/devtools-service:TraceGatherer: Waiting on DomContentLoaded
2019-06-04T18:06:50.715Z INFO @wdio/devtools-service:TraceGatherer: Waiting on DomContentLoaded
2019-06-04T18:06:50.715Z INFO @wdio/devtools-service:TraceGatherer: Waiting on DomContentLoaded
2019-06-04T18:06:51.654Z INFO @wdio/devtools-service:TraceGatherer: Found 5 inflight network records
2019-06-04T18:06:51.654Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/img/json160.gif to finish
2019-06-04T18:06:51.654Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/object.gif to finish
2019-06-04T18:06:51.654Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/value.gif to finish
2019-06-04T18:06:51.654Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/string.gif to finish
2019-06-04T18:06:51.654Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/number.gif to finish
2019-06-04T18:06:51.655Z INFO @wdio/devtools-service:TraceGatherer: Found 5 inflight network records
2019-06-04T18:06:51.655Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/img/json160.gif to finish
2019-06-04T18:06:51.655Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/object.gif to finish
2019-06-04T18:06:51.655Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/value.gif to finish
2019-06-04T18:06:51.655Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/string.gif to finish
2019-06-04T18:06:51.655Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/number.gif to finish
2019-06-04T18:06:51.686Z INFO @wdio/devtools-service:TraceGatherer: Found 4 inflight network records
2019-06-04T18:06:51.686Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/img/json160.gif to finish
2019-06-04T18:06:51.686Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/value.gif to finish
2019-06-04T18:06:51.686Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/string.gif to finish
2019-06-04T18:06:51.687Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/number.gif to finish
2019-06-04T18:06:51.687Z INFO @wdio/devtools-service:TraceGatherer: Found 4 inflight network records
2019-06-04T18:06:51.687Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/img/json160.gif to finish
2019-06-04T18:06:51.687Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/value.gif to finish
2019-06-04T18:06:51.687Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/string.gif to finish
2019-06-04T18:06:51.687Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/number.gif to finish
2019-06-04T18:06:51.725Z INFO @wdio/devtools-service:TraceGatherer: Found 3 inflight network records
2019-06-04T18:06:51.725Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/img/json160.gif to finish
2019-06-04T18:06:51.725Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/string.gif to finish
2019-06-04T18:06:51.725Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/number.gif to finish
2019-06-04T18:06:51.726Z INFO @wdio/devtools-service:TraceGatherer: Found 3 inflight network records
2019-06-04T18:06:51.726Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/img/json160.gif to finish
2019-06-04T18:06:51.726Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/string.gif to finish
2019-06-04T18:06:51.726Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/number.gif to finish
2019-06-04T18:06:51.791Z INFO @wdio/devtools-service:TraceGatherer: Found 2 inflight network records
2019-06-04T18:06:51.792Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/img/json160.gif to finish
2019-06-04T18:06:51.792Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/string.gif to finish
2019-06-04T18:06:51.813Z INFO @wdio/devtools-service:TraceGatherer: Found 1 inflight network records
2019-06-04T18:06:51.813Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/string.gif to finish
2019-06-04T18:06:51.843Z INFO @wdio/devtools-service:TraceGatherer: Found 0 inflight network records
2019-06-04T18:06:51.850Z INFO @wdio/devtools-service:TraceGatherer: Found 1 inflight network records
2019-06-04T18:06:51.851Z INFO @wdio/devtools-service:TraceGatherer: Waiting on https://www.json.org/favicon.gif to finish
2019-06-04T18:06:51.852Z INFO @wdio/devtools-service: Wait until tracing for command navigateTo finishes
2019-06-04T18:06:52.418Z INFO @wdio/devtools-service:TraceGatherer: Found 0 inflight network records
2019-06-04T18:06:56.794Z INFO @wdio/devtools-service:TraceGatherer: Network became finally idle
2019-06-04T18:06:56.794Z INFO @wdio/devtools-service:TraceGatherer: Wait for network idle canceled
2019-06-04T18:06:56.801Z INFO webdriver: COMMAND takeScreenshot()
2019-06-04T18:06:56.801Z INFO webdriver: [GET] http://127.0.0.1:9515/session/673ea9b7b70adce4e662780c094bd4e7/screenshot
2019-06-04T18:06:57.141Z INFO webdriver: RESULT iVBORw0KGgoAAAANSUhEUgAABLAAAARSCAYAAACwrQlGAAAAAXNSR0IArs4c6...
2019-06-04T18:06:57.144Z INFO webdriver: COMMAND disablePerformanceAudits()
2019-06-04T18:06:57.144Z INFO webdriver: RESULT undefined
2019-06-04T18:06:57.186Z INFO webdriver: COMMAND deleteSession()
2019-06-04T18:06:57.187Z INFO webdriver: [DELETE] http://127.0.0.1:9515/session/673ea9b7b70adce4e662780c094bd4e7
```

**Comment by christian-bromann** [comment]
@pmerwin I can't reproduce this. Did you update all packages? Also can you try to run the example here: https://github.com/christian-bromann/webdriverio-performance-testing .. I don't have any issues running performance tests there.

**Comment by pmerwin** [comment]
@christian-bromann Thanks Christian let me give it a go again... yeah I nuked node_mods etc

**Comment by pmerwin** [comment]
Still failing for me:

Here is a gist of all versions I am using:

https://gist.github.com/pmerwin/6b6642375e15f0535a115ab4969253d8


[chrome  mac os x #0-0] session.off is not a function
[chrome  mac os x #0-0] TypeError: session.off is not a function
[chrome  mac os x #0-0]     at cancel (/Users/pmerwin/Projects/qa/WDIO-5/lib-webdriverio-nodejs-v2/node_modules/@wdio/devtools-service/build/gatherer/trace.js:341:17)
[chrome  mac os x #0-0]     at Timeout.setTimeout (/Users/pmerwin/Projects/qa/WDIO-5/lib-webdriverio-nodejs-v2/node_modules/@wdio/devtools-service/build/gatherer/trace.js:297:11)
[chrome  mac os x #0-0]     at ontimeout (timers.js:482:11)
[chrome  mac os x #0-0]     at tryOnTimeout (timers.js:317:5)
[chrome  mac os x #0-0]     at Timer.listOnTimeout (timers.js:277:5)

**Comment by pmerwin** [comment]
I think the error is firing here:

/node_modules/@wdio/devtools-service/build/gatherer/trace.js
session.off('Page.domContentEventFired', domContentLoadedListener);

**Comment by pmerwin** [comment]
FYI:

When I rem out @wdio/devtools-service/build/gatherer/trace.js:341:17 this line (session.off) the test pass

**Comment by pmerwin** [comment]
I pulled down the repo:

https://github.com/christian-bromann/webdriverio-performance-testing

running npm run test:local reporduces the issue:

```
[chrome 74.0.3729.169 Mac OS X #0-1] 1) My Performance Test "before all" hook
[chrome 74.0.3729.169 Mac OS X #0-1] session.off is not a function
[chrome 74.0.3729.169 Mac OS X #0-1] TypeError: session.off is not a function
[chrome 74.0.3729.169 Mac OS X #0-1]     at cancel (/Users/pmerwin/Projects/qa/webdriverio-performance-testing/node_modules/@wdio/devtools-service/build/gatherer/trace.js:341:17)
[chrome 74.0.3729.169 Mac OS X #0-1]     at Timeout.setTimeout (/Users/pmerwin/Projects/qa/webdriverio-performance-testing/node_modules/@wdio/devtools-service/build/gatherer/trace.js:297:11)
[chrome 74.0.3729.169 Mac OS X #0-1]     at ontimeout (timers.js:482:11)
[chrome 74.0.3729.169 Mac OS X #0-1]     at tryOnTimeout (timers.js:317:5)
[chrome 74.0.3729.169 Mac OS X #0-1]     at Timer.listOnTimeout (timers.js:277:5)
```

**Comment by pmerwin** [comment]
@christian-bromann fyi ^

**Comment by pmerwin** [comment]
same issue with chrome 75:

 "spec" Reporter:
------------------------------------------------------------------
[chrome 75.0.3770.80 Mac OS X #0-0] Spec: /Users/pmerwin/Projects/qa/webdriverio-performance-testing/tests/resource.test.js
[chrome 75.0.3770.80 Mac OS X #0-0] Running: chrome (v75.0.3770.80) on Mac OS X
[chrome 75.0.3770.80 Mac OS X #0-0]
[chrome 75.0.3770.80 Mac OS X #0-0] pageWeight
[chrome 75.0.3770.80 Mac OS X #0-0]    ✖ "before all" hook
[chrome 75.0.3770.80 Mac OS X #0-0]
[chrome 75.0.3770.80 Mac OS X #0-0] 1 failing (7.2s)
[chrome 75.0.3770.80 Mac OS X #0-0]
[chrome 75.0.3770.80 Mac OS X #0-0] 1) pageWeight "before all" hook
[chrome 75.0.3770.80 Mac OS X #0-0] session.off is not a function
[chrome 75.0.3770.80 Mac OS X #0-0] TypeError: session.off is not a function
[chrome 75.0.3770.80 Mac OS X #0-0]     at cancel (/Users/pmerwin/Projects/qa/webdriverio-performance-testing/node_modules/@wdio/devtools-service/build/gatherer/trace.js:341:17)
[chrome 75.0.3770.80 Mac OS X #0-0]     at Timeout.setTimeout (/Users/pmerwin/Projects/qa/webdriverio-performance-testing/node_modules/@wdio/devtools-service/build/gatherer/trace.js:297:11)
[chrome 75.0.3770.80 Mac OS X #0-0]     at ontimeout (timers.js:482:11)
[chrome 75.0.3770.80 Mac OS X #0-0]     at tryOnTimeout (timers.js:317:5)
[chrome 75.0.3770.80 Mac OS X #0-0]     at Timer.listOnTimeout (timers.js:277:5)
------------------------------------------------------------------
[chrome 75.0.3770.80 Mac OS X #0-1] Spec: /Users/pmerwin/Projects/qa/webdriverio-performance-testing/tests/performance.test.js
[chrome 75.0.3770.80 Mac OS X #0-1] Running: chrome (v75.0.3770.80) on Mac OS X
[chrome 75.0.3770.80 Mac OS X #0-1]
[chrome 75.0.3770.80 Mac OS X #0-1] My Performance Test
[chrome 75.0.3770.80 Mac OS X #0-1]    ✖ "before all" hook
[chrome 75.0.3770.80 Mac OS X #0-1]
[chrome 75.0.3770.80 Mac OS X #0-1] 1 failing (7.7s)
[chrome 75.0.3770.80 Mac OS X #0-1]
[chrome 75.0.3770.80 Mac OS X #0-1] 1) My Performance Test "before all" hook
[chrome 75.0.3770.80 Mac OS X #0-1] session.off is not a function
[chrome 75.0.3770.80 Mac OS X #0-1] TypeError: session.off is not a function
[chrome 75.0.3770.80 Mac OS X #0-1]     at cancel (/Users/pmerwin/Projects/qa/webdriverio-performance-testing/node_modules/@wdio/devtools-service/build/gatherer/trace.js:341:17)
[chrome 75.0.3770.80 Mac OS X #0-1]     at Timeout.setTimeout (/Users/pmerwin/Projects/qa/webdriverio-performance-testing/node_modules/@wdio/devtools-service/build/gatherer/trace.js:297:11)
[chrome 75.0.3770.80 Mac OS X #0-1]     at ontimeout (timers.js:482:11)
[chrome 75.0.3770.80 Mac OS X #0-1]     at tryOnTimeout (timers.js:317:5)
[chrome 75.0.3770.80 Mac OS X #0-1]     at Timer.listOnTimeout (timers.js:277:5)

**Comment by christian-bromann** [comment]
Fixed in https://github.com/webdriverio/webdriverio/pull/4043

**Comment by pmerwin** [comment]
```
+ @wdio/cli@5.10.7
+ @wdio/devtools-service@5.10.6
+ @wdio/local-runner@5.10.7
+ webdriverio@5.10.7
+ wdio-chromedriver-service@5.0.2
```
ran npm update
nuked node_modules, did npm install ... looks fixed :) . Thanks!

**Comment by pratimghosh76** [comment]
any one able to resolve the issue. I also cloned $ git clone git@github.com:christian-bromann/webdriverio-performance-testing.git
 But getting in to error
[0-0] (node:90883) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'promise' of undefined
    at TraceGatherer.onLoadEventFired (/Users/pxg8q0v/git/webdriverio-performance-testing/node_modules/@wdio/devtools-service/build/gatherer/trace.js:117:67)
    at /Users/pxg8q0v/git/webdriverio-performance-testing/node_modules/puppeteer-core/lib/cjs/vendor/mitt/src/index.js:47:62
    at Array.map (<anonymous>)
    at Object.emit (/Users/pxg8q0v/git/webdriverio-performance-testing/node_modules/puppeteer-core/lib/cjs/vendor/mitt/src/index.js:47:43)
    at CDPSession.emit (/Users/pxg8q0v/git/webdriverio-performance-testing/node_modules/puppeteer-core/lib/cjs/puppeteer/common/EventEmitter.js:72:22)
    at CDPSession._onMessage (/Users/pxg8q0v/git/webdriverio-performance-testing/node_modules/puppeteer-core/lib/cjs/puppeteer/common/Connection.js:225:18)
    at Connection._onMessage (/Users/pxg8q0v/git/webdriverio-performance-testing/node_modules/puppeteer-core/lib/cjs/puppeteer/common/Connection.js:101:25)
    at WebSocket.<anonymous> (/Users/pxg8q0v/git/webdriverio-performance-testing/node_modules/puppeteer-core/lib/cjs/puppeteer/common/WebSocketTransport.js:13:32)
    at WebSocket.onMessage (/Users/pxg8q0v/git/webdriverio-performance-testing/node_modules/ws/lib/event-target.js:125:16)
    at WebSocket.emit (events.js:327:22)
[0-0] (node:90883) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:90883) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by mikrivorot** [comment]
Same for me
```
DevTools listening on ws://127.0.0.1:60510/devtools/browser/da2d38c2-6d76-4b5e-9529-76dd6cffe157
[0-0] (node:99656) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'promise' of undefined
```
Cannot use  `browser.enablePerformanceAudits()` with ay configuration
e.g. 
```
    await browser.enablePerformanceAudits({
      networkThrottling: 'online',
      cpuThrottling: 0,
      cacheEnabled: true
    });
```

**Comment by mgrybyk** [comment]
Try following https://github.com/webdriverio/webdriverio/actions/runs/301986390 as far as boilerplate repos might be not up to date
Use the latest webdriverio and chromedriver.

**Comment by christian-bromann** [comment]
I think

> [0-0] (node:90883) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'promise' of undefined

Is a new issue that should be reported as such.

**Comment by christian-bromann** [comment]
See #5951

**Comment by pmerwin** [comment]
FYI: I reproduced this using: 

https://github.com/christian-bromann/webdriverio-performance

2020-10-12T15:52:01.609Z ERROR @wdio/sync: TypeError: Cannot read property 'start' of undefined

Appears to be the same issue here =>

https://github.com/webdriverio/webdriverio/issues/5951:

**Comment by cg-tester** [comment]
@christian-bromann 
in my case, if I execute npm run wdio, DevTools listening on ws://127.0.0.1: ... but testing browser opens and immediately close itself. and FAILED in chrome error occurs. Could you explain about it for me please ?

**Comment by christian-bromann** [comment]
> Could you explain about it for me please ?

Without any logs or reproducible example I can't.

**Comment by cg-tester** [comment]
@christian-bromann  This is log. pls help me.

PS D:\WORKING\TestRepo\ui-automation-crmgrow> npm run wdio5

> wdio5
> wdio run ./wdio.conf.js --suite leadHub


Execution of 1 workers started at 2023-08-28T01:27:18.288Z

[0-0] RUNNING in chrome - file:///D:/WORKING/TestRepo/ui-automation-crmgrow/test/specs/login.js, file:///D:/WORKING/TestRepo/ui-automation-crmgrow/test/specs/login.js, file:///D:/WORKING/TestRepo/ui-automation-crmgrow/test/specs/leadHub.js, file:///D:/WORKING/TestRepo/ui-automation-crmgrow/test/specs/leadHub.js, file:///D:/WORKING/TestRepo/ui-automation-crmgrow/test/specs/logout.js, file:///D:/WORKING/TestRepo/ui-automation-crmgrow/test/specs/logout.js

DevTools listening on ws://127.0.0.1:59508/devtools/browser/0eae56af-aa62-4a24-9f96-cfd8f59cdf17

DevTools listening on ws://127.0.0.1:59514/devtools/browser/87897249-9f7f-4093-8b4b-2f8d3b0e25d3

DevTools listening on ws://127.0.0.1:59508/devtools/browser/0eae56af-aa62-4a24-9f96-cfd8f59cdf17

DevTools listening on ws://127.0.0.1:59514/devtools/browser/87897249-9f7f-4093-8b4b-2f8d3b0e25d3

DevTools listening on ws://127.0.0.1:59522/devtools/browser/95d6b1d0-9aa0-495a-83a4-aa3401cb1f4c

DevTools listening on ws://127.0.0.1:59527/devtools/browser/12654c00-3d47-4bcc-a9fb-35cfabb3809b
[0-0] FAILED in chrome - file:///D:/WORKING/TestRepo/ui-automation-crmgrow/test/specs/login.js, file:///D:/WORKING/TestRepo/ui-automation-crmgrow/test/specs/login.js, file:///D:/WORKING/TestRepo/ui-auton-crmgrow/test/specs/login.js, file:///D:/WORKING/TestRepo/ui-automation-crmgrow/test/specs/leadHub.js, file:///D:/WORKING/TestRepo/uics/leadHub.js, file:///D:/WOest/specs/logout.js, file:///D:/WORKIN-automation-crmgrow/test/specs/leadHub.js, file:///D:/WORKING/TestRepo/ui-automation-crmgrow/test/specs/logout.js, file:///D:/WORKING/TestRepo/ui-automation-crmgrow/test/specs/logout.js

Spec Files:      0 passed, 1 failed, 1 total (100% completed) in 00:00:07

PS D:\WORKING\TestRepo\ui-automation-crmgrow>

**Comment by cg-tester** [comment]
@christian-bromann 
and if I want to ask this kind of questions to you and your team, is there another way without using discord ?
because of some reasons, I can't use discord. maybe no skype or slack channel ?

**Comment by christian-bromann** [comment]
@cg-tester the best thing you can do is provide a minimal reproducible example on GitHub. These logs aren't the wdio logs I asked for, only your terminal logs. Please understand the we (the maintainer) can't do personal consulting hours.
